src/doc/trpl/ownership.md

```diff
@@ -121,9 +121,9 @@ let v2 = v;
 ```
 
 The first line creates some data for the vector on the [stack][sh], `v`. The
```

Comment:
I revised it, also applying your suggestions. I tried to make it as clear and explicit as possible.